addons/editor-compact/addon.json

```diff
@@ -12,6 +12,12 @@
       "link": "https://scratch.mit.edu/users/Maximouse/"
     }
   ],
+  "info": [
+    {
+      "id": "hide-icons-update",
+      "text": "To hide menu bar icons, use the \"Customizable menu bar\" addon."
```

Comment:
After #6104 the addons could be linked instead.

Reply:
It's to tell users that the behavior of Compact editor was moved to Customizable menu bar.